pkg/deploy-manager/common.go

```diff
@@ -11,9 +11,13 @@ import (
 
 // CreateNamespace creates a namespace in the cluster, ignoring if it already exists
 func (t *DeployManager) CreateNamespace(namespace string) error {
+	label := make(map[string]string)
+	// Label required for monitoring this namespace
+	label["openshift.io/cluster-monitoring"] = "true"
 	ns := &k8sv1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: namespace,
+			Name:   namespace,
```

Comment:
i can see why this is confusing and I'll make a pr to alter this a bit to make it more obvious. 

The actual deploy-with-olm yaml file is actually generated in ```pkg/deploy-manager/subscription.go```. That's why this isn't working with ocs-operator-ci.  Look for the ```generateClusterObjects()``` function

Reply:
Thanks. Updated.
Keeping the labels here so that monitoring will work on test setups.